### PR TITLE
Fix localized date header

### LINKED ISSUE #53: localize date header
For me, the current date isn't aligned with the day in datepicker.
Seems like today is thursday the 15. of november :) on locale('de')
I guess it's due to a localized `getDayOfWeek`, but non localized getDaysOfWeek (the week is starting on sunday instead of  monday).

I'll work on a fix
wrong_day

### PR DESCRIPTION
**Pull request by sakulstra**
- [x] I have changed my target branch to **develop** :facepunch:

- closes #53

## Description
yay

### CODE REVIEW

File: src/DatePicker/CalendarHeader.jsx [hunk 1]
```diff
@@ -14,6 +14,7 @@ export const CalendarHeader = (props) => {
 
   const selectNextMonth = () => onMonthChange(currentMonth.clone().add(1, 'months'));
   const selectPreviousMonth = () => onMonthChange(currentMonth.clone().subtract(1, 'months'));
+  const weekdays = [0, 1, 2, 3, 4, 5, 6].map(dow => moment().weekday(dow).format('dd'));
```
Comment (dmtrKovalenko) [inline src/DatePicker/CalendarHeader.jsx]: Please change variable name to english word.

Comment (sakulstra) [reply src/DatePicker/CalendarHeader.jsx]: @dmtrKovalenko sorry for the stupid question, but where's the "non-english" part? are you refering to dow which should be dayOfWeek in your opinion?

Comment (dmtrKovalenko) [reply src/DatePicker/CalendarHeader.jsx]: Oh, yep i have misslead the dow with foreign word meaning "day", please can you change it to dayOfWeek for example.
Thanks

**Comment by sakulstra** [comment]
I'm not at work any more - feel free to update my pr or I'll do at one of
the coming days.

Am 15.11.2017 18:16 schrieb "Dmitriy Kovalenko" <notifications@github.com>:

**Comment by dmtrKovalenko** [comment]
I will merge your PR and rename the variable possibly on weekend. Thanks for you contributing 😄

**Comment by sakulstra** [comment]
@dmtrKovalenko no worries i'm back online :) will do it today

**Pull request merged**